.circleci/config.yml

```diff
@@ -155,37 +153,13 @@ commands:
           steps:
             - run:
                 name: Export application default credentials
-                working_directory: ~/workbench
```

Comment:
Most times, this call is not necessary because `workbench-executor` executor has defined `working_directory: ~/workbench`.